nginx/nginx_test.go

```diff
@@ -268,6 +271,14 @@ func TestNginxConfig(t *testing.T) {
 				"access_log /nginx-access-log/access.log upstream_info buffer=32k flush=1m;",
 			},
 		},
+		{
+			"Access logs use custom headers when enabled",
+			logHeadersConf,
+			[]string{
+				"$http_Content-Type",
+				"$http_Authorization",
```

Comment:
Can we check the entire line, so we make sure it's part of the access log?

So
```golang
[]string {
  "'$request $status $http_content-Type $http_Authorization $body_bytes_sent",
}
```